scipy/signal/tests/test_filter_design.py

```diff
@@ -893,6 +893,14 @@ def test_nyquist(self):
         assert_array_almost_equal(w1, w2)
         assert_array_almost_equal(h1, h2)
 
+    # https://github.com/scipy/scipy/issues/17289
+    # https://github.com/scipy/scipy/issues/15273
+    def test_17289(self):
```

Comment:
It would be good to `@pytest.mark.parametrize` this over `whole`, `worN`, and `nyquist`